sapi/phpdbg/phpdbg_prompt.c

```diff
@@ -1712,6 +1712,11 @@ void phpdbg_execute_ex(zend_execute_data *execute_data) /* {{{ */
 		}
 #endif
 
+		if (exception && zend_is_unwind_exit(exception)) {
+			/* Restore bailout based exit. */
+			zend_bailout();
```

Comment:
Why does phpdbg still need the bailout?

Reply:
Because I have no idea how to handle it properly ... maybe @bwoebi knows?

Reply:
Do we need special handling at all here?